Module not found: Error: Can't resolve '@datorama/akita/lib/runStoreAction'

## I'm submitting a...

<!-- Check one of the following options with "x" -->
<pre><code>
[x] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

If you import the runStoreAction like in the docs [here](https://datorama.github.io/akita/docs/additional/events/)
you get this error:
`'Module '"../../../../../node_modules/@datorama/akita/datorama-akita"' has no exported member 'EntityStoreAction'.ts`
and if you import like VSCode suggestes:
```
import {getEntityType } from '@datorama/akita';
import { StoreAction, runEntityStoreAction, EntityStoreAction, runStoreAction } from '@datorama/akita/lib/runStoreAction';
```
you get: `Module not found: Error: Can't resolve '@datorama/akita/lib/runStoreAction'`
## Environment

<pre><code>
Angular version: 9.0.1

</code></pre>

--- a/libs/akita/src/lib/index.ts
+++ b/libs/akita/src/lib/index.ts
@@ -55,7 +55,7 @@ export { SelectAllOptionsA, SelectAllOptionsB, SelectAllOptionsC, SelectAllOptio
 export { __stores__ } from './stores';
 export { isDev, enableAkitaProdMode, __DEV__ } from './env';
 export { isNotBrowser } from './root';
-export { runStoreAction, runEntityStoreAction, getStore, getEntityStore } from './runStoreAction';
+export { runStoreAction, runEntityStoreAction, getStore, getEntityStore, EntityStoreAction, StoreAction } from './runStoreAction';
 export { arrayUpdate } from './arrayUpdate';
 export { arrayAdd } from './arrayAdd';
 export { arrayUpsert } from './arrayUpsert';

